src/accordion/accordion.js

```diff
@@ -0,0 +1,109 @@
+// @flow
```

Comment:
[nit] Does this file need the copyright header?

Reply:
it does! let me fix it